index.html

```diff
@@ -15,11 +15,16 @@
         <h1 class='fb-name'>facebook</h1>
         <div class='inner-form'>
             <h4>Log in to Facebook</h4>
-            <input type='text' placeholder='Email address or phone number' class='email_or_phone'>
-            <input type='password' placeholder='Password' class='password'>
-            <input type='submit' value='Log in' class='login'>
+            <input type='text' placeholder='Email address or phone number' id='email-or-phone' class='email_or_phone'>
+            <input type='password' placeholder='Password' id='password' class='password'>
+            <a href='' class='btn login-btn' type='button'>Log in</a>
+            <div class='mobile-mode'>
```

Comment:
This div only work media query max width  **780px** with, 

- [ ] Mobile responsive